resource/opentelemetry-resource-detector-azure/src/opentelemetry/resource/detector/azure/app_service.py

```diff
@@ -44,26 +46,28 @@
 class AzureAppServiceResourceDetector(ResourceDetector):
     def detect(self) -> Resource:
         attributes = {}
-        website_site_name = environ.get(_WEBSITE_SITE_NAME)
-        if website_site_name:
-            attributes[ResourceAttributes.SERVICE_NAME] = website_site_name
-            attributes[
-                ResourceAttributes.CLOUD_PROVIDER
-            ] = CloudProviderValues.AZURE.value
-            attributes[
-                ResourceAttributes.CLOUD_PLATFORM
-            ] = CloudPlatformValues.AZURE_APP_SERVICE.value
-
-            azure_resource_uri = _get_azure_resource_uri(website_site_name)
-            if azure_resource_uri:
+        try:
+            website_site_name = environ.get(_WEBSITE_SITE_NAME)
```

Comment:
I think generic error handling is already addressed on the Resource side -> https://github.com/open-telemetry/opentelemetry-python/blob/main/opentelemetry-sdk/src/opentelemetry/sdk/resources/__init__.py#L397